tmux/PKGBUILD

```diff
@@ -1,40 +1,37 @@
 # Maintainer: Alexey Pavlov <alexpux@gmail.com>
 
-pkgname=tmux-git
-pkgver=2.5.94.g73b9328c
+pkgname=tmux
+pkgver=2.6
 pkgrel=1
 pkgdesc='A terminal multiplexer'
 url='https://tmux.github.io/'
 arch=('i686' 'x86_64')
 license=('BSD')
 provides=('tmux')
```

Comment:
It cannot provide itself, remove this line

Reply:
@mati865 Fixed